edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -1074,6 +1097,18 @@ export default {
       return this.$store.getters['i18n/withFallback'](`cluster.addonChart."${ name }"`, null, fallback);
     },
 
+    showAddons() {
+      this.addonsRev++;
+      this.addonNames.forEach((name) => {
+        if (this.versionInfo[name]?.questions) {
+          this.userChartValuesTemp[name] = this.initYamlEditor(name);
+        } else {
+          this.userChartValuesTemp[name] = {};
```

Comment:
These need to be `set(this.userChartValuesTemp, name, …)` or the key on the object won't be reactive, which is what cause your problem with Questions disappearing on blur/rerender.

Reply:
Changed!